pkg/test/network.go

```diff
@@ -1,14 +1,22 @@
 package test
 
-import "net"
+import (
+	"fmt"
+	"net"
+)
 
 func GetFreePort() (int, error) {
-	ln, err := net.Listen("tcp", ":0")
+	port, err := FindFreePort("")
+	return int(port), err
+}
+
+func FindFreePort(ip string) (uint32, error) {
```

Comment:
It's a little bit weird to have `Get()` that returns `int` and `Find()`. Can we try to get rid of the `Get` with `int`?

Reply:
`GetFreePort` doesn't have an argument, `FindFreePort` does

Reply:
I see, ok then